src/views/Project/index.jsx

```diff
@@ -57,6 +57,30 @@ const tagReposMapping = repositories =>
       ...mappings,
     };
   }, {});
+const extractWhiteboardTags = whiteboard => {
```

Comment:
This kind of function just screams for unit tests, doesn't it?  I don't think there's any setup for that in codetribute, though..